administrator/modules/mod_content/mod_content.xml

```diff
@@ -16,9 +16,9 @@
 	</files>
 	<languages>
 		<language tag="en-GB">en-GB.mod_content.ini</language>
-		<language tag="en-GB">en-GB.mod_latest.sys.ini</language>
+		<language tag="en-GB">en-GB.mod_content.sys.ini</language>
 	</languages>
-	<help key="JHELP_EXTENSIONS_MODULE_MANAGER_ADMIN_LATEST" />
+	<help key="JHELP_EXTENSIONS_MODULE_MANAGER_ADMIN_CONTENT" />
```

Comment:
Damit ichs nicht vergesse: Das müsste dann auch in der en-GB.ini und auf dem JDocs Server hinzugefügt werden. (JHELP_EXTENSIONS_MODULE_MANAGER_ADMIN_LATEST)